### PR TITLE
Add default license header to files which have no copyright or license header yet

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Follow-up to #5065.

CC @gotmax23

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
various

### CODE REVIEW

File: tests/integration/targets/gitlab_project_members/aliases [hunk 1]
```diff
@@ -1 +1,5 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
 unsupported
```
Comment (gotmax23) [inline tests/integration/targets/gitlab_project_members/aliases:5]: We might as well fix this (no new line at end of file) while we're already touching the file...

Comment (felixfontein) [reply tests/integration/targets/gitlab_project_members/aliases:5]: Let's better do this in a follow-up PR. There are hundrets of files that have this, and some might already be touched in this PR while others are not.

Comment (felixfontein) [reply tests/integration/targets/gitlab_project_members/aliases:5]: https://stackoverflow.com/a/44747386 seems to be helpful to do this for a large number of files at once.

**Comment by felixfontein** [comment]
@gotmax23 thanks for taking a look at this one as well!

**Comment by felixfontein** [comment]
Manual backport in #5075.

File: tests/integration/targets/ldap_search/tasks/run-test.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (gotmax23) [inline tests/integration/targets/ldap_search/tasks/run-test.yml:4]: Not related to licensing at all, but is this task file suppose to be empty?

Comment (felixfontein) [reply tests/integration/targets/ldap_search/tasks/run-test.yml:4]: Fixed in #5077.

**Pull request merged**